packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -385,14 +391,33 @@ export class LitNodeClientNodeJs
         `Storage key "${storageKey}" is missing. Not a problem. Contiune...`
       );
 
+      // Check if a valid session key exists in cache
+      if (
+        sessionKeyCache &&
+        Date.now() - sessionKeyCache.timestamp < TEMP_CACHE_PERIOD
```

Comment:
i don't understand this.  why is the session key time sensitive, with a 30s cache period?  like it should be cached until the inner auth sig delegation sig thingy expires?

Reply:
Do you mean using the included `expiration` in the `siwe` as an expiration time? If so i think this makes sense.

Reply:
yeah, like that's something i would look at.  the SIWE expiration.  but i don't understand where 30s came from?  they're good until the SIWE expiration?

Reply:
I thought 30s should be enough to finish any operations you have requested, anything more than that you should request a new session key unless you want to store it yourself.

Reply:
I think it makes sense to use the SIWE expiration date